cmd/ethkey/changepassphrase.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"gopkg.in/urfave/cli.v1"
+)
+
+var commandChangePassphrase = cli.Command{
+	Name:      "changepassphrase",
+	Usage:     "change the passphrase on a keyfile",
+	ArgsUsage: "<keyfile>",
+	Description: `
+Change the passphrase of a keyfile.`,
+	Flags: []cli.Flag{
+		passphraseFlag,
+		cli.StringFlag{
+			Name:  "newpassfile",
```

Comment:
I see that you renamed the existing flag to match this one. `--newpasswordfile` sounds bad, but it's probably the better option so we don't break existing scripts.